src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -1478,6 +1479,7 @@ export default class DefaultDeviceController
     const oldDevice = this.activeDevices[kind];
 
     this.activeDevices[kind] = newDevice;
+    this.logger.info(`Set activeDevice to ${JSON.stringify(newDevice)}`);
```

Comment:
Do we need this to print out everytime? If not we may want to change it to debug level.